homeassistant/components/minecraft_server/config_flow.py

```diff
@@ -60,33 +71,73 @@ def _show_config_form(self, user_input=None, errors=None) -> FlowResult:
             errors=errors,
         )
 
-    async def _async_is_server_online(self, address: str) -> bool:
-        """Check server connection using a 'status' request and return result."""
+    async def _async_is_java_server_online(self, address: str) -> bool:
```

Comment:
I know it's because of debug log lines, but this is a lot of lines for essentially 10 lines of code.

Reply:
You mean that I have to many debug logs? I could remove the good case logs if you want?

Reply:
Obsolete --> Please re-check with latest commit.